node/src/components/consensus/highway_core/active_validator.rs

```diff
@@ -602,14 +605,14 @@ mod tests {
         }
 
         /// Handle new unit by validator `vidx`.
-        /// Since all validator use the same state, that unit should be added already. Panics if
+        /// Since all validators use the same state, that unit should be added already. Panics if
         /// not. Returns effect created when handling new unit.
         fn handle_new_unit(
             &mut self,
             vidx: ValidatorIndex,
             uhash: &<TestContext as Context>::Hash,
         ) -> Vec<Effect<TestContext>> {
-            let validator = self.validators.get_mut(&vidx).unwrap();
+            let validator = self.active_validators.get_mut(vidx);
```

Comment:
It already implements `IndexMut`, so probably this would work, without a new method?
```suggestion
            let validator = &mut self.active_validators[vidx];
```

Reply:
Done in [`e8c11ff` (#602)](https://github.com/CasperLabs/casper-node/pull/602/commits/e8c11fff198245dfcb56564f615cdc6ed5848ac4).